### PR TITLE
Ana/4917 add parameter to polkadot democracy (API)

### PR DESCRIPTION
**Pull request by Bitcoinera**
Related to #4917
**Description:**

The API part needed to get parameter in Proposal (Polkadot democracies)

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
@faboweb do you know a better way to convert from hex to ASCII? I get some strange unicode stuff in between:

image

**Comment by faboweb** [comment]
Polkadot API doesn't provide a method to deserialize?

**Comment by Bitcoinera** [comment]
> Polkadot API doesn't provide a method to deserialize?

Yes, I was looking into this. Will keep looking

**Comment by Bitcoinera** [comment]
Don't merge, still fixing some things here. Need to debug

**Review by jbibla — DISMISSED** [review]

File: api/lib/source/polkadotV0-source.js [hunk 1]
```diff
@@ -1047,6 +1047,20 @@ class polkadotAPI {
     }
   }
 
+  getProposalParameter(proposal) {
+    // democracy proposals in Polkadot contain a parameter which is what is going to be changed
+    if (proposal.image) {
+      const imageProposal = JSON.parse(JSON.stringify(proposal.image.proposal))
+      if (imageProposal.args.old) {
+        return `Old Ethereum Address ${imageProposal.args.old} | New Ethereum Address ${imageProposal.args.new}`
```
Comment (jbibla) [inline api/lib/source/polkadotV0-source.js:1055]:
```suggestion
        return `Old: ${imageProposal.args.old} | New: ${imageProposal.args.new}`
```

**Review by jbibla — APPROVED** [review]

**Pull request merged**